torch/csrc/jit/ir.cpp

```diff
@@ -516,6 +529,15 @@ void Block::cloneFrom(Block * src, std::function<Value*(Value*)> value_map) {
     local_map[input] = this->addInput()->copyMetadata(input)->setStage(input->stage());
     graph->setStage(std::max(graph->stage(), input->stage()));
   }
+  const auto entryWorld = src->entryWorld();
+  if (!entryWorld->outputs().empty()) {
```

Comment:
These conditionals mean that this code gets tested very rarely. One of the reasons I want it non-conditional is that we find bugs in how cloning and other code interacts with the world token handling early.